src/components/utils/chartUtils.ts

```diff
@@ -78,15 +78,24 @@ export const getChartOptions = (input: {
   const extendedCategories =
     categories.length > 0 ? [categories[0] - 1, ...categories] : []
 
+  const xaxisCategories =
+    categories.length > 0
+      ? [
+          categories[0] - 1,
+          ...categories.slice(0, -1),
+          `${categories[categories.length - 1]}+`,
+        ]
```

Comment:
Kan eventuelt vurdere å ta utgangspunkt i `extendedCategories` i stedet, slik at du slipper å gjøre `categories[0] - 1` to ganger.